src/pages/LoginPage.js

```diff
@@ -1,16 +1,102 @@
-import React from "react";
-import { Link } from "react-router-dom";
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import "../styles/LoginPage.css";
 
-function LoginPage() {
+export default function Login() {
+  const [formData, setFormData] = React.useState({ email: "", password: "" })
+  const [item, setItems] = React.useState([])
+  const [formErrors, setFormErrors] = React.useState({});
+  const history = useNavigate();
+
+
+  React.useEffect(function () {
+    fetch('https://jsonplaceholder.typicode.com/users')
+      .then((response) => response.json())
+      .then(data => setItems(data))
+  }, [])
+
+
+  function handleChange(event) {
+    setFormData(prevFormData => {
+      return {
+        ...prevFormData,
+        [event.target.name]: event.target.value
+      };
+    });
+  }
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    setFormErrors(validate(formData));
+  };
+
+  const validate = (values) => {
+    const errors = {};
+    const userid = formData.email;
+    const passid = formData.password;
+
+
+
+    item.map((items) => {
```

Comment:
got to switch there, you are mapping `items` and picking each `item` from the array, not the other way around